lib/galaxy/datatypes/proteomics.py

```diff
@@ -72,7 +72,7 @@ def display_peek(self, dataset):
 class ProtXmlReport(Tabular):
     """protxml converted to tabular report"""
     edam_data = "data_2536"
-    file_ext = "tsv"
+    file_ext = "protxml.tsv"
```

Comment:
@jmchilton I was also assuming it has something do to with the download extension and I think there should be such a `reall-ife` extension defined in a datatype, isn't it?